src/checkbox/Checkbox.ts

```diff
@@ -141,7 +141,7 @@ export default class Checkbox extends CheckboxBase<CheckboxProperties> {
 
 		if (label) {
 			checkboxWidget = w(Label, {
-				classes: this.classes(css.root, ...stateClasses),
+				extraClasses: this.classes(css.root, ...stateClasses),
```

Comment:
As this is going to be called in each instance that a `label` widget is used, I think that `parseLabelClasses` should be a function exported from `label`.